src/Uno.UWP/Devices/Midi/MidiInPort.wasm.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.Devices.Enumeration.Internal;
+
+namespace Windows.Devices.Midi
+{
+    public partial class MidiInPort
+    {
+		private const string JsType = "Windows.Devices.Midi.MidiInPort";
+		private readonly string _wasmId;
+
+		private MidiInPort(string deviceId, string wasmId)
+		{
+			DeviceId = deviceId;
+			_wasmId = wasmId;
+		}
+
+		public void Dispose()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either implement 'IDisposable.Dispose', or totally rename this method to prevent confusion.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)